docs/deployment/advanced-configuration.rst

```diff
@@ -136,6 +137,67 @@ There are several default values that are attached to each created metadata. If
         https://purl.org/fair-metrics/FM_F1A: https://www.ietf.org/rfc/rfc3986.txt
         https://purl.org/fair-metrics/FM_A1.1: https://www.wikidata.org/wiki/Q8777
 
+FDP Index
+=========
+
+You can turn your FAIR Data Point instance into a FDP Index that can be contacted by other FDPs and harvests metadata from them.
+
+Hosting FDP Index
+-----------------
+
+To enable FDP Index mode on your FDP server, just simply adjust your ``application.yml`` file:
+
+.. code:: yaml
+    
+    # application.yml
+
+    fdp-index:
+        enabled:  true
+
+
+Then for the FDP client, you need to use ``fairdata/fairdatapoint-index-client`` Docker image for browsing indexed FDPs and searching harvested metadata. In case you want to use your deployment both as FDP and FDP Index, you can deploy both FDP and FDP Index client applications. The configuration of both clients is identical.
```

Comment:
small typo `both clients is identical` to `both clients are identical`

Reply:
fixed